samples/app-with-targetframeworks/MultipleTargetFrameworks/MultipleTargetFrameworks.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.2</TargetFrameworks>
```

Comment:
I'd recommend using 2.1 instead of 2.2 here as 2.2 is out of support.

Reply:
oO my bad, forgot to change that before pushing ^^

Reply:
(done)